modules/tinymce/CHANGELOG.md

```diff
@@ -8,23 +8,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 6.6.0 - TBA
 
-### Fixed
-- Fixed an issue that prevented setting the inline dialog `size` to 'medium' or 'normal'. #TINY-10015
-
 ### Added
-- Added new `bottom` to inline dialog type. This new inline dialog option allows the inline dialog to be positioned at the bottom of the editor. #TINY-9888
-- Added new `persistent` option to `WindowParams`. This new option allows the inline dialog to be persistent, meaning that it will not be closed when the user clicks outside of the dialog. #TINY-9991
+- Added a new property — `bottom` — for inline dialog configurations. #TINY-9888
+- Added a new property — `persistent` — for inline dialog configurations that controls dialog persistence when not in focus. #TINY-9991
```

Comment:
I have a higher standard of docs than "turning on setting x controls the feature x behaviour". It's like saying the sky is blue because it has a blue colour.

```suggestion
- Added a new property — `persistent` — for inline dialog configurations that will stop the dialog closing when clicking away from it. #TINY-9991
```